src/command_modules/azure-cli-sf/azure/cli/command_modules/sf/custom.py

```diff
@@ -217,8 +225,9 @@ def print_progress(size, rel_file_path):
                 url = urlunparse(url_parsed)
 
                 def file_chunk(target_file, rel_path, print_progress):
-                    chunk = target_file.read(100000)
-                    if chunk != b'':
+                    chunk = True
+                    while chunk:
+                        chunk = target_file.read(100000)
```

Comment:
A reasonable size to read into memory at one time? @jeffreychenmsft may have a better answer here.